SaltedgeAuthenticatorSDKv2/Classes/API/SEEncryptedAuthorizationData.swift

```diff
@@ -23,32 +23,34 @@
 import Foundation
 import SEAuthenticatorCore
 
-public struct SEEncryptedAuthorizationData: SEBaseEncryptedAuthorizationData, SerializableResponse {
+public struct SEEncryptedAuthorizationData: SEBaseEncryptedAuthorizationData, Decodable {
     public let id: String
     public let data: String
     public let key: String
     public let iv: String
     public let status: AuthorizationStatus
     public var connectionId: String?
-    public var entityId: String?
+    public var entityId: String? //TODO: check if we need it
```

Comment:
Did you checked? This variable comes from `SEBaseEncryptedAuthorizationData` protocol. If you want, you could find another way of using it. If no - rm todo.